src/test/java/org/hibernate/infra/bot/tests/PullRequestOpenedTest.java

```diff
@@ -76,7 +77,6 @@ void titleEndsWithEllipsis() throws IOException {
 									❌ The pull request title should not end with an ellipsis (make sure the title is complete)
 
 									› This message was automatically generated.""" );
-					verifyNoMoreInteractions( mocks.ghObjects() );
```

Comment:
Is this necessary? Why?

Reply:
This check always failed after creating the new editing action, I guess because each action ("check pull request" and "edit pull request" respectively) interacts with the mock separately? I'm not that experienced with Mockito though, so I can't say for sure.

Reply:
Ok, but there's a point to these checks, so I'd prefer keeping them.

Maybe if you move the block "if ( repositoryConfig == null || repositoryConfig.enableEditing == null || repositoryConfig.enableEditing.equals( Boolean.FALSE )" to the *top* of `editPullRequestBodyAddIssueLinks`, before the call to `shouldCheck`, the new "action" won't affect the mocks when it's disabled anymore?

Reply:
Yeah that did the trick, thanks for the tip! Updating the PR now with your suggestions, note that I'll also change the README to briefly mention this new feature as I forgot to do that before.